lib/plugins/utils.js

```diff
@@ -47,14 +47,25 @@ export function validateStep({required}, conf) {
   return conf.length === 0 || validateSteps(conf);
 }
 
-export async function loadPlugin({cwd}, name, pluginsPath) {
+export async function loadPlugin({ cwd }, name, pluginsPath) {
   const basePath = pluginsPath[name]
     ? dirname(resolveFrom.silent(__dirname, pluginsPath[name]) || resolveFrom(cwd, pluginsPath[name]))
     : __dirname;
 
   // See https://github.com/mysticatea/eslint-plugin-node/issues/250
```

Comment:
i think this comment can be removed at this point

Reply:
```suggestion
```